serverless/networking/serverless-ossm-custom-domains.adoc

```diff
@@ -1,4 +1,4 @@
-include::modules/serverless-document-attributes.adoc[]
+poinclude::modules/serverless-document-attributes.adoc[]
```

Comment:
Is poinclude a typo? Never seen this before :smile:

Reply:
Definitely a typo, darn it!  =P